### PR TITLE
Change actions/github-script from e69ef54 -> 60a0d83

### PR DESCRIPTION
**Pull request by Vertexwahn**
Change actions/github-script from e69ef54 -> 60a0d83

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         if [ -s fmt.patch ]; then
           exit 1
         fi
-    - uses: actions/github-script@e69ef5462fd455e02edcaf4dd7708eda96b9eda0 # v7
+    - uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7
```
Comment (vitaut) [inline .github/workflows/lint.yml:32]: Please update the version to v7.0.1 to keep it in sync with the hash.

**Comment by vitaut** [comment]
LGTM (modulo version update) but is there any specific reason for bumping the version?

**Comment by Vertexwahn** [comment]
@vitaut No. On my side renovate complained about on old version

**Comment by Vertexwahn** [comment]
@vitaut Sorry, I was wrong. Renovate found more actions to update - will update the PR once renovate is done ;)

**Comment by Vertexwahn** [comment]
Only lint and cifuzz were not up to date...

**Comment by vitaut** [comment]
thanks

**Pull request merged**